packages/sdk/src/mb-impl/constants.ts

```diff
@@ -0,0 +1,41 @@
+export enum METHOD_NAMES {
+  'TRANSFER' = 'nft_transfer',
+  'BATCH_TRANSFER' = 'nft_batch_transfer',
+  'BATCH_BURN' = 'nft_batch_burn',
+  'DEPLOY_TOKEN_CONTRACT' = 'create_store',
+  'TRANSFER_TOKEN_CONTRACT_OWNERSHIP' = 'transfer_store_ownership',
+  'MINT' = 'nft_batch_mint',
+  'ADD_MINTER' = 'grant_minter',
+  'REMOVE_MINTER' = 'revoke_minter',
+  'BATCH_CHANGE_MINTERS' = 'batch_change_minters',
+  'TOKEN_ACCOUNT_REVOKE' = 'nft_revoke',
+  'TOKEN_ACCOUNT_REVOKE_ALL' = 'nft_revoke_all',
+}
+
+export enum MARKET_METHOD_NAMES {
+  'LIST' = 'nft_approve',
+  'DEPOSIT_STORAGE' = 'deposit_storage',
+  'BUY' = 'buy',
+}
+
+export enum GAS_CONSTANTS {
```

Comment:
Why not just use regular constant string values for these?  Enumerated objects are good for referencing values by index (number) in a list form, but purely for organization/collection cases where only a few values are needed or just using strings in general seems overkill

Reply:
*Applies to other enums in here as well I think, they aren't really lists of a single field

Reply:
Updated these.